README.md

```diff
@@ -102,6 +112,16 @@ To add more replicas you can use the juju scale-application functionality i.e.
 juju scale-application temporal-worker-k8s <num_of_replicas_required_replicas>
 ```
 
+## Error Monitoring
+
+The Temporal worker operator has a built-in Sentry interceptor which can be used
+to intercept and capture errors from the Temporal SDK. To enable it, run the
+following command:
+
+```bash
+juju config temporal-worker-k8s sentry-dsn=<YOUR_SENTRY_DSN>
```

Comment:
There is an envvar for Sentry `SENTRY_RELEASE` IIRC, that lets you version your releases so that Sentry can track lineage of errors between versions. It is not mandatory but it might be a nice functionality here.

Reply:
Updated to include Sentry release and environment configurations. Thanks!